tasks/prime.py

```diff
@@ -374,31 +374,18 @@ def update_mto_shipment_status(self):
     @tag(PrimeObjects.MOVE_TASK_ORDER.value, "createMoveTaskOrder")
     @task
     def create_move_task_order(self):
-        payload = {
+
+        overrides = {
+            # We need these objects to exist
+            "destinationDutyStationID": "71b2cafd-7396-4265-8225-ff82be863e01",
+            "originDutyStationID": "1347d7f3-2f9a-44df-b3a5-63941dd55b34",
+            "uploadedOrdersID": "c26421b0-e4c3-446b-88f3-493bb25c1756",
             "contractorId": "5db13bb4-6d29-4bdb-bc81-262f4513ecf6",
-            "moveOrder": {
-                "customer": {
-                    "firstName": "Christopher",
-                    "lastName": "Swinglehurst-Walters",
-                    "agency": "MARINES",
-                    "email": "swinglehurst@example.com",
-                    "rank": "E_3",
-                    "dodID": "4586736251",
-                },
-                "entitlement": {"nonTemporaryStorage": False, "totalDependents": 47},
-                "orderNumber": "32",
-                "rank": "E_3",
-                "tac": "F8J1",
-                "destinationDutyStationID": "71b2cafd-7396-4265-8225-ff82be863e01",
-                "originDutyStationID": "1347d7f3-2f9a-44df-b3a5-63941dd55b34",
-                "uploadedOrdersID": "c26421b0-e4c3-446b-88f3-493bb25c1756",
-                "ordersType": "GHC",
-                "reportByDate": "2020-01-01",
-                "status": "SUBMITTED",
-                "issueDate": "2020-01-01",
-            },
-            "status": "SUBMITTED",
+            # Currently the state engine will fail on creating all mto service items
+            # unless the state is APPROVED
+            "status": "APPROVED",
```

Comment:
Was the issue you ran into that this status was applied to both the `move` and the `moveOrder`? I think I re-understand the way it works now (it's been a while since I wrote this 😅) - you can nest overrides like you do in a payload. So the above could also be written as:

```python
overrides = {
        "status": "APPROVED",
        "moveOrder": {
            "status": "APPROVED",
            "destinationDutyStationID": "71b2cafd-7396-4265-8225-ff82be863e01",
            "originDutyStationID": "1347d7f3-2f9a-44df-b3a5-63941dd55b34",
            "uploadedOrdersID": "c26421b0-e4c3-446b-88f3-493bb25c1756",
        }
}
```

The upside being that that is more explicit about what's being overridden, the downside being that you can't shortcut the setting of "status" anymore. You have to explicitly set that override for the "moveOrder" too. 

So these are the two ways overrides can be used - set ANY field with that name to a certain value, or explicitly set the values in nested objects. I think if you did something like `"moveOrder": {}` it would actually generate fake data for this object without repeating any of the overrides from the top level (but I didn't test that to be sure). Man, I sure overdid it with this hahaha

Reply:
Also, I believe you don't need to set the `moveTaskOrder` "status" field to APPROVED - the `makeAvailableToPrime` endpoint should also approve the MTO. But either way is fine with me.

Reply:
Oh I see, I need to nest it. I guess what threw me off was that it DID show up in the moveOrder but it also showed up in every other object. Like just, everything hehe. Let me see if I can get it to work exactly right.